awx/api/serializers.py

```diff
@@ -126,7 +126,7 @@
     'current_job': DEFAULT_SUMMARY_FIELDS + ('status', 'failed', 'license_error'),
     'inventory_source': ('source', 'last_updated', 'status'),
     'custom_inventory_script': DEFAULT_SUMMARY_FIELDS,
-    'source_script': ('name', 'description'),
+    'source_script': DEFAULT_SUMMARY_FIELDS,
```

Comment:
The API changes here are pretty innocuous.  Including the `id` for `source_script` will have little performance impact, and if it saves the UI a request to API, then I think it is worth it.